config/backup_schedule.rb

```diff
@@ -0,0 +1,3 @@
+every :day, :at => '1:00am' do
```

Comment:
Maybe we can use the new syntax here, what do you think? `every :day, at: '1:00am' do`